ui/component/or-dashboard-builder/src/widgets/or-gauge-widget.ts

```diff
@@ -190,7 +190,7 @@ export class OrGaugeWidgetSettings extends LitElement {
                             ></or-mwc-input>
                         </div>
                         <div>
-                            <or-mwc-input .type="${InputType.NUMBER}" style="width: 100%;" .value="${config.decimals}" label="${i18next.t('decimals')}"
+                            <or-mwc-input type="${InputType.NUMBER}" style="width: 100%;" .value="${config.decimals}" label="${i18next.t('decimals')}" .min="${0}"
```

Comment:
This now prevents people from using negative values.

Reply:
I did because I don't know what -2 decimals means. That 100 becomes 1? 🙃